Johny1010/WDP-1909-01
# Wdp190901 32

# Description
- index.html: added link to the new section
- new section :80-section-feedback.html:
new container with img + personal info and feedback
- new scss : section--feedback.scss: style for new section

## Related Jira issue

https://projects.kodilla.com/browse/WDP190901-32?filter=myopenissues
## Steps to test
- check hover and active dots
- check if all info from jira is included 

# Checklist


- [X] I have manually tested my code.
- [X] I have reorganised my commits to make them readable and easily reversible.
- [ ] If this is a UI change, provide a screen-shot or video demo:

# A gif(t) describing how this change makes you feel (optional)
![X] https://www.google.com/search?tbm=isch&sxsrf=ACYBGNSamOeiEwNE9lGAyi-wk3_uC_7klg%3A1570652425928&sa=1&ei=CUGeXYihOKmFk74PyJCNqAM&q=happy+gif&oq=happy+gif&gs_l=img.3..35i39j0l3j0i30l6.4480.4617..5014...0.0..0.105.198.1j1......0....1..gws-wiz-img.......0i5i30.qpsj9o6c7Yw&ved=0ahUKEwjIyaafgJDlAhWpwsQBHUhIAzUQ4dUDCAc&uact=5#imgrc=uXPWydY0I7ibNM:
